docs/errors/index.md

```diff
@@ -17,19 +17,24 @@ Some error codes occur more frequently than others. Here is a list of the common
 | --- | --- |
 | [ERR\_NGROK\_100](/docs/errors/err_ngrok_100) | Invalid metadata length |
 | [ERR\_NGROK\_108](/docs/errors/err_ngrok_108) | You've hit your account limit for simultaneous ngrok agent sessions. Try stopping an existing agent or upgrading your account. |
-| [ERR\_NGROK\_1226](/docs/errors/err_ngrok_1226) | You are disallowed from creating an ngrok account due to violation of the terms of service. |
-| [ERR\_NGROK\_4100](/docs/errors/err_ngrok_4100) | The email or password you entered is not valid. |
+| [ERR\_NGROK\_120](/docs/errors/err_ngrok_120) | Your ngrok agent version "<VERSION>" is no longer supported. Only the most recent version of the ngrok agent is supported without an account. Update to a newer version with `ngrok update` or by downloading from https://ngrok.com/download. Sign up for an account to avoid forced version upgrades: https://ngrok.com/signup. |
```

Comment:
```suggestion
| [ERR\_NGROK\_120](/docs/errors/err_ngrok_120) | Your ngrok agent version "\<VERSION\>" is no longer supported. Only the most recent version of the ngrok agent is supported without an account. Update to a newer version with `ngrok update` or by downloading from https://ngrok.com/download. Sign up for an account to avoid forced version upgrades: https://ngrok.com/signup. |
```